### PR TITLE
Added new property: VoiceConnection#gateway

### PR DESCRIPTION
**Pull request by SenpaiWeb**
**Please describe the changes this PR makes and why it should be merged:**
This is useful if you need the gateway the guild is connected to

**Status**
- [X] Code changes have been tested against the Discord API, or there are no code changes
- [ ] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [X] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by SpaceEEC** [comment]
When would you need that gateway? (Any example?)

This would also need documentation.
Initialize the property to null in the constructor and add a doc-string there.

**Comment by SenpaiWeb** [comment]
@SpaceEEC, I had to use it to automatically switch server region when the gateway packet loss was > 10%, and for the documentation lemme add it

**Comment by SenpaiWeb** [comment]
yes, because it won't give you the current endpoint of the region such as: `us-westXXX.discord.gg`

**Comment by appellation** [comment]
The specific node of the voice region seems to be irrelevant if you're completely switching region.

File: src/client/voice/VoiceConnection.js [hunk 1]
```diff
@@ -51,6 +51,12 @@ class VoiceConnection extends EventEmitter {
      */
     this.status = VoiceStatus.AUTHENTICATING;
 
+    /**
+    * Get the current gateway the guild is linked to
```
Comment (vladfrangu) [inline src/client/voice/VoiceConnection.js]:
```suggestion
    * Get the current gateway this voice connection is linked to
```

**Comment by amishshah** [comment]
I don't really see a point in this, sorry

**Pull request closed without merging**